<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContactDialog</class>
 <widget class="QDialog" name="AddContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>251</width>
     <height>346</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>имя</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="firstNameEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>фамилия</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lastNameEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>отчество</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="middleNameEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>адрес</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addressEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>email </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="emailEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>дата рождения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="birthDateEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>телефон</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phoneEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddContactDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddContactDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
